script.js

```diff
@@ -114,14 +145,21 @@
 // ====================================================================
 // Write your solution below.
 // ====================================================================
-
+/*
 var testArray = [1, 2, 4, 8, 16, 32, 64, 128];
-
-
-
-
-
-
+function shuffle(a) {
+    var j, x, i;
+    for (i = a.length - 1; i > 0; i--) {
+        j = Math.floor(Math.random() * (i + 1));
+        x = a[i];
+        a[i] = a[j];
+        a[j] = x;
+    }
+    return a;
+}
+testArray=shuffle(testArray);
+console.log(testArray);
+*/
```

Comment:
Try to use more descriptive variable names as a matter of good coding practice. Code readability is important when working in teams, and any increase in file size is completely negated by pre-production compilers/optimizers.